docs/guides/endgame.md

```diff
@@ -507,10 +508,10 @@ already exist in the project. These updates are reserved for
 8. Manually check to see that the `latest` tags were updated for the packages
    correctly by going to [`npm`](https://www.npmjs.com), searching the packages,
    viewing the versions and making sure the `latest` version matches the version
-   you just released.
```

Comment:
hmmm I think this one is more about time proximity than 'its as simple as', although I don't know if removing it substantially changes the sentance.